learning-bitmark/working-with-bitmarks/using-bitmark-shares.md

```diff
@@ -0,0 +1,631 @@
+# Bitmark shares
+
+In some cases, the ownership of a property is shared between different parties or people. To support those cases, the Bitmark Property System provides a feature called the **[bitmark shares]()**.
+
+Any owner of a Bitmark Certificate is able to:
+
+* [Create bitmark shares](#creating-bitmark-shares) from that certificate (or bitmark).
+
+* [Grant bitmark shares](#granting-bitmark-shares-to-another-account) to another account.
+
+* [Swap bitmark shares](#swapping-bitmark-shares) with others.
+
+
+Currently, only the Bitmark CLI supports performing those transactions of bitmark shares.
+
+> **NOTE:** Transactions related to the bitmark share require a transaction fee. On the bitmark network, the fee can be pay by BTC or LTC
+>
+> * Real BTC and/or LTC are used to pay for transaction fee on the Bitmark main chain
+>
+> * Testnet BTC and/or LTC are used to pay for transaction fee on the Bitmark testing chain
+
+<br>
+<br>
+
+## Bitmark shares related records
+
+There are three records related to the Bitmark shares are stored on the Bitmark blockchain
+
+* *Balance Record* - created by a bitmark owner to permanently set the total number of a particular share for the bitmark.
+
+* *Grant Record* - created by an owner to grant an amount of his share balance to another owner.
+
+* *Swap Record* - created by two owners to swap their shares of different bitmarks together.
+
+<br>
+<br>
+
+## Prerequisites of working with bitmark shares using the Bitmark CLI
+
+Refer the [Payment on Bitmark CLI](payment-on-bitmark-cli.md) document to:
+
+* Install the Bitmark-Wallet for transaction payments.
+
+<br>
+<br>
+
+## Creating bitmark shares
+
+Any bitmark owner is able to create shares of a particular bitmark, that action divides the bitmark into a number of shares
+
```

Comment:
When we have a section that then has subsections, it's best to clarify & organize things by talking about what the subsections will do. For this section, it might be something like:

"Any bitmark owner can divide a Bitmark certificate into a number of shares by creating the shares, then paying the associated fees. Additional commands can be used to verify the transaction."

Reply:
Updated.